src/inspect_ai/tool/_tools/_web_browser/_resources/playwright_crawler.py

```diff
@@ -285,10 +285,12 @@ def render(self, output_format: CrawlerOutputFormat) -> Any:
           the currently active webpage rendered using given format.
         """
         match output_format:
+            case CrawlerOutputFormat.HTML:
+                return self._page.content()
```

Comment:
One other thought, we limit tool output to 16kb to go easy on context windows (if you have tool calls that frequently exceed this you will both fill up the context window and sometimes confuse the model with spurious content). One reason to favor accessibility trees is that they are much smaller and semantically clear so better as model input.